ex1/exceptions.py

```diff
@@ -46,3 +62,13 @@
 # the first by the second. This function should handle exceptions that might
 # occur print out what went wrong and return None if no results could be
 # computed.
+
+# if in the try exceptions are raised the "except" block is called
+# so if for example an ZeroDivisionError occurs, None is returned
+def safe_divide(a, b):
+    try:
+        result = a / b
+    except:
```

Comment:
Excepting all possible exceptions is generally a dangerous idea. Since it hides all possible errors from you.